File: sdkwork-ai-sdk-kotlin/src/main/kotlin/com/sdkwork/ai/DocumentAddRequest.kt
package com.sdkwork.ai

data class DocumentAddRequest(
    val content: String? = null,
    val title: String? = null,
    val type: String? = null,
    val metadata: Map<String, Any>? = null,
    val chunkConfig: ChunkConfig? = null,
    val knowledgeBaseId: String? = null
)

File: sdkwork-ai-sdk-python/sdkwork_ai_sdk/models/document_add_request.py
from dataclasses import dataclass
from typing import Optional, List, Dict, Any

@dataclass
class DocumentAddRequest:
    """Document add request"""
    content: str
    title: str = None
    type: str = None
    metadata: Dict[str, Any] = None
    chunk_config: ChunkConfig = None
    knowledge_base_id: str
